src/gcodeExport.cpp

```diff
@@ -377,14 +377,27 @@ double GCodeExport::getTotalFilamentUsed(int extruder_nr)
     return extruder_attr[extruder_nr].totalFilament;
 }
 
-double GCodeExport::getTotalPrintTime()
+std::vector<double> GCodeExport::getTotalPrintTimes()
```

Comment:
Okay I renamed all of them because there were several with this name and functionality.